core/src/components/footer/footer.tsx

```diff
@@ -1,5 +1,7 @@
 import type { ComponentInterface } from '@stencil/core';
-import { Component, Element, Host, Prop, h } from '@stencil/core';
+import { Component, Element, Host, Prop, State, h } from '@stencil/core';
+import type { KeyboardController } from '@utils/keyboard/keyboard-controller';
+import { createKeyboardController } from '@utils/keyboard/keyboard-controller';
```

Comment:
We can't use relative imports in non-test code until https://github.com/ionic-team/stencil/issues/2815 is resolved.